params/config.go

```diff
@@ -178,6 +179,7 @@ var (
 		TerminalTotalDifficultyPassed: true,
 		ShanghaiTime:                  newUint64(0),
 		CancunTime:                    newUint64(0),
+		NostoiBlock:                   big.NewInt(0),
```

Comment:
Do we set it to 0 for local node to override?

Reply:
Local is for testing, so it assumes all features (e.g. block = 0 / timestamp = 0) are enabled